kotlin-react/src/main/kotlin/react/Suspense.ext.kt

```diff
@@ -0,0 +1,8 @@
+import react.Render
+import react.SuspenseProps
+import react.createElement
+
+// 16.6+
+fun SuspenseProps.fallback(handler: Render) {
+    asDynamic().fallback = createElement(handler)
```

Comment:
Redundant `asDynamic`?

Reply:
`SuspenseProps` are decalared next way now:
```
external interface SuspenseProps : PropsWithChildren
```
Shell I declare it this way:
```
external interface SuspenseProps : PropsWithChildren {
    var fallback: ReactNode?
}
```

Reply:
[Yes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b8e894eaa4ce70830a40a6bcc3c12392eb867989/types/react/index.d.ts#L400)

Reply:
fixed